iron-range.html

```diff
@@ -37,7 +40,10 @@
      * @type number
```

Comment:
These annotations can be removed. Our tools are able to automatically derive them from the `properties` definition now.